<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewprofile</class>
 <widget class="QDialog" name="viewprofile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_vpvv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="pushButton_backvp">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_editvp">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sjoblist">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>View Profile</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vppic">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_vpfp">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>109</y>
     <width>1261</width>
     <height>601</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_vpfn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vpln">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vppn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone Number</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vpad">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_vped">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Education</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fname">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>90</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_lname">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_phonenumber">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>180</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_address">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>240</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_education">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>266</y>
      <width>391</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_email">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>390</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Company</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_current">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>371</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
